README.md

```diff
@@ -69,14 +71,17 @@ public virtual ContentReference GoogleAnalyticsSettings { get; set; }
 * Assign the value of the property to the setting you created in the local settings.
 * After you have done this, content under the node that has assigned the local setting should get these settings, while content outside of this structure should get the global settings.
 
-## Removing settings classes
-Before removing settings classes it is important to delete any instances of the classes  that has been saved to the database, otherwise there will be problems accessing the folders containing settings. This can be done by going into the adminstrative interface/Tools/Manage Content. Settings instances can be found below *Root/For all Sites/Global Settings Root|Settings Root*.
+## Removing Settings classes
+If upgrading from an earlier version of AddOn.Episerver.Settings, some values must be set in the database first. In table tblContentType each row that contains a Settings-type must have the Base-column updated to the value **Setting**. This ensures that existing settings entities can be loaded by the CMS even if the type class no longer exists.
 
-1. Deploy code no longer using the obsolete classes to the site.
-2. Remove obsolete instances through the administrative interface.
-3. Remove classes from codebase, then deploy again.
+Then locate and delete all instances that are of the no longer existing settings-types. Remember to remove from the _Global Settings_ **and** to empty the trash can. After this is done the types will be dropped automatically the next time the CMS is restarted.
+
+0. Update tblContentType, set Base column to "Setting" for Settings types.
```

Comment:
Can this be done by setting the base class for all settings type in the system instead of working directly against the DB?

Reply:
I could not find a way to have system update the value. It only writes there when it has discovered  and adding new ContentTypes not with any updates. The base value is discovered by checking the inheritance provided by the ContentTypeBaseProvider so there is no property to toggle on the ContentTypes themselves.

Reply:
Ok, so this means a one-time SQL-update for current users but will not be a problem for people that start with the new version then?